### PR TITLE
Implemented C1285 (TaaS)(vega)

### PR DESCRIPTION
**Pull request by victoriakor**
https://foliotest.testrail.io/index.php?/cases/view/1285
https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/4941/allure/

### CODE REVIEW

File: cypress/e2e/title-level-request/check_cannot_change_tlr_modal.cy.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import { TestTypes, DevTeams, Permissions } from '../../support/dictionary';
+import { REQUEST_TYPES, FULFILMENT_PREFERENCES, REQUEST_LEVELS } from '../../support/constants';
+import UserEdit from '../../support/fragments/users/userEdit';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import Location from '../../support/fragments/settings/tenant/locations/newLocation';
+import Users from '../../support/fragments/users/users';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import getRandomPostfix from '../../support/utils/stringTools';
```
Comment (dnovikau) [inline cypress/e2e/title-level-request/check_cannot_change_tlr_modal.cy.js:8]: Please remove unused imports

File: cypress/support/fragments/settings/circulation/titleLevelRequests.js [hunk 2]
```diff
@@ -42,8 +47,8 @@ export default {
           cy.expect(
             Checkbox({ name: 'titleLevelRequestsFeatureEnabled', disabled: false }).exists(),
           );
-          cy.do(TLRCheckbox.click());
-          cy.do(saveButton.click());
+          this.clickOnTLRCheckbox();
```
Comment (dnovikau) [inline cypress/support/fragments/settings/circulation/titleLevelRequests.js:50]: Modifying of existing functions could lead to unexpected failures. Please do not use construnctions with 'this' keyword. 

As an alternative you can create report with execution of all tests where this method is used to be sure the this change did not affeted them.

**Review by dnovikau — CHANGES_REQUESTED** [review]

File: cypress/e2e/title-level-request/check_cannot_change_tlr_modal.cy.js (hunk 1 above)
Comment (victoriakor) [reply cypress/e2e/title-level-request/check_cannot_change_tlr_modal.cy.js]: done

File: cypress/support/fragments/settings/circulation/titleLevelRequests.js (hunk 2 above)
Comment (victoriakor) [reply cypress/support/fragments/settings/circulation/titleLevelRequests.js:50]: done (removed steps with 'this' statement)

**Comment by victoriakor** [comment]
created a new one

**Pull request closed without merging**